### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by Bober65031** [berarma/new-lg4ff#81] [Q&A]
Im total noob in linux and i dont know how to install this. Im using fedora pls dont hate me

**Comment by Norodix** [comment]
This is a kernel module which means it needs to be build for your exact kernel version. Because of that it is not distributed as a binary but as source code and you need to build it. 

There is a helper program for that called dkms (Dynamic Kernel Module System). dkms will build it for you and put it in the correctl place so that it works with your system.

There is a pretty detailed explanation in the readme on how to use it, you can read it here.
https://github.com/berarma/new-lg4ff#dkms
Follow these steps and if you are stuck somewhere ask.

Since you are trying something you have not done before, make sure that if you have any important data on your PC ,back it up! This is not a process that is likely to cause data loss but better safe than sorry.
